src/routes/Messages/PeopleSelector/PeopleSelector.test.js

```diff
@@ -52,7 +51,7 @@ describe('PeopleSelector', () => {
           />
         </MemoryRouter>
       )
-      peopleSelectorComponent = wrapper.find(PeopleSelector)
+      wrapper.find(PeopleSelector)
```

Comment:
hmm weird, well I guess id we are no longer using peopleSelectorComponent we dont need this line anymore either

Reply:
All in description above ^ linting stuff from a branch merged yesterday. Thinking of new devs and "verify" on `git commit` which would fail ...

Reply:
im saying we can just remove this line completely, its not doing anything

Reply:
Gotcha. Ok, I'll do that